src/styles/blocks/button.scss

```diff
@@ -0,0 +1,31 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+
+  text-decoration: none;
+  font-weight: 700;
+  color: #fff;
+  background-color: $colorDark;
+  border: 3px solid $colorDark;
+  border-radius: 40px;
+  text-align: center;
+
+  cursor: pointer;
+  transition: all 0.5s;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s
It's too hard for the browser
Just list all property 
![image](https://user-images.githubusercontent.com/91411692/224299636-d4a769f4-1417-4e7e-bb5a-5f24a6ba003f.png)
also, create variable for this
```suggestion
  transition: border-color $transitionDuration, color $transitionDuration;
```